auth/jwt.go

```diff
@@ -0,0 +1,67 @@
+package auth
+
+import (
+"github.com/dgrijalva/jwt-go"
+
+"crypto/rsa"
+"errors"
+"fmt"
+"time"
+)
+
+// todo: key rotation
```

Comment:
Try to think of some better naming here. This struct is not a JWT per se, it is a service that issues JWTs.